demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -89,7 +92,16 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
         self.walletSDK = walletSDK
         result(true)
     }
-    
+     
+    /**
+              This method gets the version detail if we build sdk using the env variable
+              For Example: NEW_VERSION=testVer GIT_REV=testRev BUILD_TIME=testTime make generate-ios-bindings copy-ios-bindings
+     */
+    public func getWalletSDKVersion(result: @escaping FlutterResult) {
+        let version = VersionGetVersion()
+        print("getting version detail ->", version)
+        result(version)
```

Comment:
Can we show Gitversion as well as Build time as well ?

https://github.com/trustbloc/wallet-sdk/blob/main/demo/app/ios/Runner/Tests/IntegrationTest.swift#L25-L26

Reply:
Yes can add that too. Will add n update